pyincore/analyses/housingrecovery/housingrecovery.py

```diff
@@ -291,7 +273,7 @@ def merge_seasonal_data(self, hse_rec, vac_status):
         vac_status["tractid"] = vac_status["state"].astype(str) + vac_status["county"].astype(str) + \
                                 vac_status["tract"].astype(str)
 
-        hse_rec_merged = pd.merge(hse_rec, vac_status, on="tractid", how="inner")
+        hse_rec_merged = pd.merge(hse_rec, vac_status, left_on="tractid", right_on="tractid", how='inner')
```

Comment:
aren't these two lines equivalent? left+right vs on for inner merge.

Just wondering if this was done to fix something

Reply:
It is the same thing. It probably stayed from data validation. There was a test dataset with tract (not tractid) on the "right" side.